harness/determined/_train_context.py

```diff
@@ -152,8 +158,13 @@ class TrialContext(_TrainContext):
     The context passed to the User's ``Trial.__init__()`` will inherit from this class.
     """
 
-    def __init__(self, env: det.EnvContext, hvd_config: horovod.HorovodContext):
-        super().__init__(env, hvd_config)
+    def __init__(
+        self,
+        env: det.EnvContext,
+        hvd_config: horovod.HorovodContext,
+        rendezvous_info: det.RendezvousInfo,
+    ) -> None:
+        super().__init__(env, hvd_config, rendezvous_info)
 
 
 class NativeContext(_TrainContext):
```

Comment:
Shouldn't this be removed? I thought we have already removed the native API.

Reply:
No, it is not removed.  It is only deprecated.  I have stated that I would like to remove `det.experimental.keras.init()` and `det.experimental.estimator.init()` for the `0.16.0` release, but that won't happen until right before the `0.16.0` release.